docs/api/client-request.md

```diff
@@ -47,6 +47,7 @@ following properties:
     be aborted. When mode is `manual` the redirection will be cancelled unless
     [`request.followRedirect`](#requestfollowredirect) is invoked synchronously
     during the [`redirect`](#event-redirect) event.  Defaults to `follow`.
+  * `origin` String (optional) - The origin URL of the request.
```

Comment:
Should we perhaps only allow setting this through the headers?

Reply:
We need to set the origin url but not the origin header for sometime.